Exception/FlattenException.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Elao\ErrorNotifierBundle\Exception;
+
+class FlattenException
+{
+    /** Symfony 2.3+ */
+    const DEBUG_FLATTEN     = 'Symfony\Component\Debug\Exception\FlattenException';
+    /** Symfony <2.3 (deprecated in 2.3) */
+    const CONFIG_FLATTEN    = 'Symfony\Component\Debug\Exception\FlattenException';
```

Comment:
Yeah that would be correct. I thing I just copied and pasted the line above and forgot to update it.